pkgs/top-level/all-packages.nix

```diff
@@ -25283,6 +25283,10 @@ with pkgs;
     inherit (plasma5Packages) breeze-icons;
   };
 
+  gruvbox-material-icons-gtk = callPackage ../data/icons/gruvbox-material-icons-gtk {
+    inherit (plasma5Packages) breeze-icons;
```

Comment:
I think `plasma5Packages` shouldn't be used here.

Reply:
I assumed, as breeze-icons sits in the scope of libsForQt5 and I noticed others are using plasma5Packages for breeze-icons too.

Reply:
Fair enough. I was just thinking that it being an icon set, it should be DE-agnostic.